src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -199,6 +199,10 @@ public function type($path)
      */
     public function mimeType($path)
     {
+        if (!extension_loaded("fileinfo")) {
```

Comment:
Your code does not comply expected style, please add https://styleci.io/analyses/874pEX